js/main.js

```diff
@@ -69,7 +80,17 @@ function dataEntry(name, image, id) {
 }
 
 function getRecipe(event) {
-  var detailedId = event.target.dataset.id;
+
+  while ($recipeIngredients.firstChild) {
+    $recipeIngredients.removeChild($recipeIngredients.firstChild);
+  }
+  while ($recipeInstructions.firstChild) {
+    $recipeInstructions.removeChild($recipeInstructions.firstChild);
+  }
```

Comment:
Instead of repeating this behavior all over your app, you should make a reusable function where you can pass an argument of the element you want to delete the children of.